addOns/spider/src/main/java/org/zaproxy/addon/spider/SpiderDialog.java

```diff
@@ -212,7 +212,7 @@ private void setAdvancedTabs(boolean visible) {
 
     @Override
     public String getHelpIndex() {
-        return "ui.dialogs.spider";
+        return "addon.spider";
```

Comment:
`addon.spider.dialog`

Reply:
Yo be clear when I said tab and options weren’t working, I meant within the help itself, not on the panels. 

Yes I’ll fix these.